### ISSUE TITLE
How can I build image for python3.9 runtime?

### ISSUE THREAD

**Issue by mhihasan** [lambci/docker-lambda#349]
I am using python3.9 runtime, and that's why I need `lambci/lambda:build-python3.9` image. As currently it is not available, I would like to build from myself and use this until any officail release for python3.9. 

How could I do that?
Thanks

**Comment by senor-coder** [comment]
AWS has a docker image for python 3.9 ->  public.ecr.aws/lambda/python:3.9 . Does this solve your problem ?

**Comment by mhihasan** [comment]
This image requires requires the handler name to be the first argument of entrypoint, but I want to build lambda layer using that,

```bash
docker run --rm  --volume "$PWD:/var/task" --workdir /var/task  public.ecr.aws/lambda/python:3.9 pip install -Ur requirements.txt --target .layers/demo/python
```
This command throws error,
```bash
entrypoint requires the handler name to be the first argument
```

**Comment by senor-coder** [comment]
@mhihasan You can try out the images that I have published for python 3.9 in the meantime.

`docker run --rm  --volume "$PWD:/var/task" --workdir /var/task  senorcoder/aws-lambda-env:python3.9_build pip install -Ur requirements.txt --target .layers/demo/python`

```
Runtime: senorcoder/aws-lambda-env:python3.9
Build  : senorcoder/aws-lambda-env:python3.9_build
```

Repository Reference:
https://github.com/senor-coder/aws-lambda-pyruntime

**Comment by mhihasan** [comment]
Thanks a lot, it works. 👍
